tools/python/stylesheet/color_matrix_builder.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python2.7
```

Comment:
Please write a good doc string about sequence of you steps. With details and not obvious things